src/components/Board/index.tsx

```diff
@@ -11,25 +25,147 @@ const BoardContainer = styled.div`
   padding: 20px;
 `;
 
-const QuestionTile = styled.div`
-  background-color: #007bff;
+/* const QuestionTile = styled.div` */
+/*   background-color: #007bff; */
+/*   color: white; */
+/*   padding: 20px; */
+/*   text-align: center; */
+/*   cursor: pointer; */
+/*   transition: background-color 0.2s ease-in-out; */
+
+/*   &:hover { */
+/*     background-color: #0056b3; */
+/*   } */
+/* `; */
+
+// 
+const CategoriesContainer = styled.div`
+  display: grid;
+  grid-template-columns: repeat(5, 1fr);
+  gap: 10px;
+  padding: 20px;
+`;
+
+const CategoryTile = styled.div`
+  background-color: #0056b3;
   color: white;
   padding: 20px;
   text-align: center;
-  cursor: pointer;
-  transition: background-color 0.2s ease-in-out;
-
-  &:hover {
-    background-color: #0056b3;
-  }
 `;
 
 export const Board = () => {
+  const [isAnswerShown, setIsAnswerShown] = useState<boolean>(false);
+  const [selectedQuestion, setSelectedQuestion] = useState<Question | null>(
+    null
+  );
+  const [team1Score, setTeam1Score] = useState<number>(
+    Number(localStorage.getItem("team1Score")) || 0
+  );
+  const [team2Score, setTeam2Score] = useState<number>(
+    Number(localStorage.getItem("team2Score")) || 0
+  );
+  const [team1Clicked, setTeam1Clicked] = useState<boolean>(false);
+  const [team2Clicked, setTeam2Clicked] = useState<boolean>(false);
+
+  // handle scores; toggles between addition and resetting 
+  const handleTeam1Correct = () => {
+    if (!team1Clicked) {
+      setTeam1Score((prevScore) => prevScore + selectedQuestion!.value);
+      setTeam1Clicked(true);
+    } else {
+      setTeam1Score((prevScore) => prevScore - selectedQuestion!.value);
+      setTeam1Clicked(false);
+    }
+  };
+
+  const handleTeam2Correct = () => {
+    if (!team2Clicked) {
+      setTeam2Score((prevScore) => prevScore + selectedQuestion!.value);
+      setTeam2Clicked(true);
+    } else {
+      setTeam2Score((prevScore) => prevScore - selectedQuestion!.value);
+      setTeam2Clicked(false);
+    }
+  };
```

Comment:
You could refactor this to one component like this: 

`const handleTeam2Correct = () => {
    if (!team2Clicked) {
      setTeam2Score((prevScore) => prevScore + selectedQuestion!.value);
      setTeam2Clicked(true);
    } else {
      setTeam2Score((prevScore) => prevScore - selectedQuestion!.value);
      setTeam2Clicked(false);
    }
  };`